src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -365,18 +354,29 @@ public void clickSuConfermaElimina() {
         this.getWebDriverWait(30).withMessage("Non è stato possibile cliccare sul bottone conferma").until(ExpectedConditions.elementToBeClickable(confermaRimuoviPECBy));
         this.element(confermaRimuoviPECBy).click();
     }
+    public boolean siControllaEliminazionePEC() {
+        try {
+            this.getWebDriverWait(10).until(ExpectedConditions.visibilityOf(pecField));
+            logger.info("pec non presente");
+            return true;
+        } catch (TimeoutException e) {
+            logger.error("pec presente con errore" + e.getMessage());
+            return false;
+        }
```

Comment:
I think you can minimize the code just by checking if it is displayed or not
```suggestion
        return pecField.isDisplayed();
```